epos_library/include/epos_library/Definitions.h

```diff
@@ -59,15 +57,16 @@
     Initialisation_DllExport int  VCS_CloseAllDevices(unsigned int* pErrorCode);
 
 //Info
-    HelpFunctions_DllExport int  VCS_GetVersion(void* KeyHandle, unsigned short NodeId, unsigned short* pHardwareVersion, unsigned short* pSoftwareVersion, unsigned short* pApplicationNumber, unsigned short* pApplicationVersion, unsigned int* pErrorCode);
-    HelpFunctions_DllExport int  VCS_GetErrorInfo(unsigned int ErrorCodeValue, char* pErrorInfo, unsigned short MaxStrSize);
     HelpFunctions_DllExport int  VCS_GetDriverInfo(char* p_pszLibraryName, unsigned short p_usMaxLibraryNameStrSize,char* p_pszLibraryVersion, unsigned short p_usMaxLibraryVersionStrSize, unsigned int* p_pErrorCode);
+	HelpFunctions_DllExport int  VCS_GetVersion(void* KeyHandle, unsigned short NodeId, unsigned short* pHardwareVersion, unsigned short* pSoftwareVersion, unsigned short* pApplicationNumber, unsigned short* pApplicationVersion, unsigned int* pErrorCode);

```

Comment:
the whole Definitions.h file is copied from the official library releasey by maxonmotor. I did not do any modifications -> I don't feel responsible or even see the need to style this --> thanks to maxonmotor for messing up the style and me receiving the blame. 

See https://www.maxonmotor.de/medias/sys_master/root/8825412976670/EPOS-Linux-Library-En.zip